plutus-core/testlib/PlutusIR/Generators/QuickCheck/ShrinkTerms.hs

```diff
@@ -122,6 +124,10 @@ minimalBuiltin (SomeTypeIn uni) = case toSingKind uni of
     go (DefaultUniProtoList `DefaultUniApply` _)                        = []
     go (DefaultUniProtoPair `DefaultUniApply` a `DefaultUniApply` b)    = (go a, go b)
     go (f  `DefaultUniApply` _ `DefaultUniApply` _ `DefaultUniApply` _) = noMoreTypeFunctions f
+    go DefaultUniBLS12_381_G1_Element                                   = BLS12_381.G1.zero
+    go DefaultUniBLS12_381_G2_Element                                   = BLS12_381.G2.zero
+    go DefaultUniBLS12_381_MlResult                                     = error "No representation for MlResult elements"
```

Comment:
Are you saying here that there's absolutely no way to construct a `BLS12_381_MlResult`? That doesn't sound right. Like, you're supposed to return _something_ here, it doesn't need to be something specific, just literally anything. Can be a result of some function or whatever.

Reply:
I guess we could pair the zero elements for G1 and G2 to get the identity element in `MlResult`.  I'll give that a try.